LICENSE

```diff
@@ -1,4 +1,4 @@
-node-graylog (C) 2011 Egor Egorov <me@egorfine.com> 
+MIT License
```

Comment:
I have changed this, since there's literally nothing left of the original implementation.